samples/highcharts/website/static-editor/demo.js

```diff
@@ -7712,10 +7712,10 @@ highed.DataTable = function (parent, attributes) {
             value: getVal,
             destroy: destroy,
             addToDOM: addToDOM,
-            selectCell: selectCell,
+            selectrenderTo: selectCell,
             deleteContents: deleteContents,
-            deselectCell: deselectCell,
-            deselectCopyCell: deselectCopyCell,
+            deselectrenderTo: deselectCell,
+            deselectCopyrenderTo: deselectCopyCell,
```

Comment:
IMO it shouldn't be changed.

Reply:
I fixed it in many places, but I missed this reference. Thanks!